erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -198,11 +198,11 @@ def setup_ageing_columns(filters: Filters, range_columns: List):
 		f"0 - {filters['range1']}",
 		f"{cint(filters['range1']) + 1} - {cint(filters['range2'])}",
 		f"{cint(filters['range2']) + 1} - {cint(filters['range3'])}",
-		f"{cint(filters['range3']) + 1} - {_('Above')}",
+		_("{0} - Above").format(cint(filters['range3']) + 1),
 	]
```

Comment:
Doing this way are more control to translation, example in spanish i would translate this as "90 días a más" or "mayor a 90 días"